astropy/config/tests/data/suppress.cfg

```diff
@@ -0,0 +1,2 @@
+[utils.data]
+dataurl = http://example.org/astropy/suppress_config/
```

Comment:
It is just a test if the feature works, it could also be any other configuration variable. This was just the first example i found in the other tests.